### PR TITLE
nsh: Add tests for NSH headers types 1 and 2

### PR DESCRIPTION
**Pull request by carlgsmith**
To be used for once NSH header support is accepted into Suricata

### CODE REVIEW

**Comment by catenacyber** [comment]
https://github.com/OISF/suricata/pull/5309

File: tests/decode-nsh-type1/test.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# *** Add configuration here ***
```
Comment (catenacyber) [inline tests/decode-nsh-type1/test.yaml:1]: You can remove this header line

**Review by catenacyber — COMMENTED** [review]
This looks good to me.
You can add a README.md file to each test, so as to explain its purpose

File: tests/decode-nsh-type1/test.yaml (hunk 1 above)
Comment (carlgsmith) [reply tests/decode-nsh-type1/test.yaml:1]: > You can remove this header line

@catenacyber Do I need something this in test.yaml as well?

`requires:
    min-version: 6
`

Comment (catenacyber) [reply tests/decode-nsh-type1/test.yaml:1]: Yes indeed, as NSH is not yet in version 5

**Comment by carlgsmith** [comment]
Updated by https://github.com/OISF/suricata-verify/pull/307

**Pull request closed without merging**